src/Buttons/__tests__/BorderlessButton.test.tsx

```diff
@@ -65,4 +71,47 @@ describe('Borderless Button', function () {
       assert.strictEqual(clickSpy.callCount, 0);
     });
   });
+  context('when forwardRef prop is present', function () {
+    beforeEach(function () {
+      const ButtonRefExample = () => {
+        const ref = useRef<HTMLInputElement>(null);
+        const onButtonClick = () => {
+          setTimeout(() => {
+            if (ref.current) {
+              ref.current.focus();
+            }
+          });
+        };
+        return (
+          <>
+            <Button
+              onClick={onButtonClick}
+              variant={VARIANT.PRIMARY}
+            >
+              Focus the Borderless Button
+            </Button>
+            <BorderlessButton
+              onClick={() => {}}
+              variant={VARIANT.DANGER}
+              tabIndex={0}
+              forwardRef={ref}
+              alt="Danger Button"
```

Comment:
Trivial change : instead of `Danger Button` I prefer something like `Borderless Button` because the child is not always a danger (trash) button.

Reply:
That's a good point - updated the naming through a5e21f1.